### PR TITLE
allow the release number to be specified manually

### PR DESCRIPTION
**Pull request by sandlerr**
@zendesk/samson @grosser @miogalang

### CODE REVIEW

**Comment by grosser** [comment]
screenshot ?

... some potential for conflict if 2 ppl open the new form at the same time ... but pretty intuitive ...

👍

File: app/views/releases/new.html.erb [hunk 1]
```diff
@@ -3,14 +3,19 @@
   <section>
     <fieldset>
       <div class="row">
-        <p class="col-lg-offset-2 col-lg-4">Manual releases should only be created when your CI service is down, and you need to release a new version regardless. The commit will tagged as <%= @release.version %>, which will be pushed to the Git repository.</p>
+        <p class="col-lg-offset-2 col-lg-4">Manual releases should only be created when your CI service is down, and you need to release a new version regardless. The commit will tagged and the tag will be pushed to the Git repository.</p>
```
Comment (craig-day) [inline app/views/releases/new.html.erb]: s/The commit will tagged/The commit will be tagged

**Comment by sandlerr** [comment]
There is a unique index that should crash in that case. The git push should probably not work either.

**Comment by sandlerr** [comment]
screen shot 2016-08-26 at 9 52 43 am

**Pull request merged**